code/src/main/java/com/google/sps/data/PositiveUser.java

```diff
@@ -0,0 +1,41 @@
+package com.google.sps.data;
+
+import com.googlecode.objectify.annotation.Entity;
+import com.googlecode.objectify.annotation.Id;
+import java.time.Instant;
+
+@Entity
+public class PositiveUser {
+
+  @Id String userId;
+  private int emailsSent = 0;
+  private long firstLogin;
+  private long lastLogin;
+
+  private PositiveUser() {}
+  
+  public PositiveUser(String userId) {
+    this.userId = userId;
+    firstLogin = lastLogin = Instant.now().getEpochSecond();
+  }
+
+  public void setLastLogin() {
+    firstLogin = Instant.now().getEpochSecond();
+  }
+  
+  public long getNumberOfEmailsSent() {
+      return emailsSent;
+  }
+
+  public long getFirstLogin() {
+    return firstLogin;
+  }
+
+  public long getLastLogin() {
+    return lastLogin;
+  }
+
```

Comment:
I feel like updateEmailsSent should have an int param and go up by said value

Reply:
I was thinking that too but I also thought we would only want to increment when an email is successfully sent?

Reply:
That actually makes sense, works for me